packages/client/src/components/Login/Testimonials.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types'; // Import PropTypes for prop validation
+import { Typography } from '@mui/material';
+import meraj from './meraj.jpg'; // Import images for each testimonial
+import yagmur from './yagmur.jpg';
+import rumana from './rumana.jpg';
+
+const testimonialsData = [
+  {
+    id: 1, // Add a unique identifier for each testimonial
+    image: meraj,
+    text: "It's so convenient to provide updates and see what everyone else is working on.",
+    author: 'Meraj Sharifi, Team Lead',
+  },
+  {
+    id: 2,
+    image: yagmur,
+    text: 'No more endless email chains or missed stand-up meetings.',
+    author: 'Yagmur Nielsen, Software Engineer',
+  },
+  {
+    id: 3,
+    image: rumana,
+    text: 'Being able to see past check-ins helps us track our progress and stay accountable.',
+    author: 'Rumana, Designer',
+  },
+];
+
+const Testimonial = ({ image, text, author }) => {
+  return (
+    <div className="testimonial-card">
+      <img className="images" src={image} alt={author} />
```

Comment:
Nice catch with the `alt` tag! It is always required to have a description for the non-visual users :)